analysis.js

```diff
@@ -71,26 +72,26 @@ function FileBuilder()
 	this.report = function()
 	{
 		console.log (
-			( "{0}\n" +
-			  "~~~~~~~~~~~~\n"+
-			  "ImportCount {1}\t" +
-			  "Strings {2}\n"
+			( '{0}\n' +
+			  '~~~~~~~~~~~~\n'+
+			  'ImportCount {1}\t' +
+			  'Strings {2}\n'
 			).format( this.FileName, this.ImportCount, this.Strings ));
 	}
 }
 
 // A function following the Visitor pattern.
 // Annotates nodes with parent objects.
-function traverseWithParents(object, visitor)
```

Comment:
Change function declaration notation from old notation to new arrow function notation